docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+version: "3.9"
+services:
+  todo-app:
+    build:
+      context: .
+      target: development
+    ports:
+      - 5000:5000
+    volumes:
+      - ./todo_app:/todo_app
```

Comment:
This isn't quite right. The right-hand side of the colon should be `/app/todo_app` - you set `WORKDIR /app` so that's the location in the container where you need to put the app files.

You can test if the bind mount is correct by starting up the container, editing a file, like index.html and checking if the change is immediately visible
Currently you have to rebuild the image to get the change, which makes it pretty inconvenient for development.